personalwebsite/projects/views.py

```diff
@@ -5,7 +5,7 @@
 # Create your views here.
 def projects(request):
     all_projects = Project.objects.all()
-    paginator = Paginator(all_projects, 3)
+    paginator = Paginator(all_projects, 6)
```

Comment:
I believe this might be causing a bug where the pagination might not work correctly, you can find the bug in the trello board for pagination.